charts/agent-control-deployment/templates/_helpers.tpl

```diff
@@ -69,6 +69,8 @@ cluster name, licenses, and custom attributes
 
 {{- /* Add to config k8s cluster and namespace config */ -}}
 {{- $k8s := (dict "cluster_name" (include "newrelic.common.cluster" .) "namespace" .Release.Namespace "namespace_agents" .Values.subAgentsNamespace) -}}
+{{- /* Add ac_remote_update and cd_remote_update to the config */ -}}
+{{- $k8s = mustMerge $k8s (dict "ac_remote_update" .Values.acRemoteUpdate "cd_remote_update" .Values.cdRemoteUpdate) -}}
```

Comment:
Why don't we add them as fields in the above line? Just curious it there's something I don't understand there.